app/controllers/budgets/investments_controller.rb

```diff
@@ -56,7 +56,10 @@ def new
     def show
       @commentable = @investment
       @comment_tree = CommentTree.new(@commentable, params[:page], @current_order)
-      @related_contents = Kaminari.paginate_array(@investment.relationed_contents).page(params[:page]).per(5)
+      related_without_retired_proposals = @investment.relationed_contents
+                                                      .select { |rel| rel[:retired_at] == nil }
```

Comment:
In order to avoid duplication, do you think we could move this line to the `Relationable` concern? Maybe in the `relationed_contents` itself :thinking:; that way we won't have to change every place using it (like the `DashboardController`) and probably the change in the view won't be necessary either :thinking:.

Reply:
One more thing: using `rel[:retired_at]` is certainly an interesting idea :ok_hand:. However, it isn't something we usually use. We usually write something like: `.reject { |related| related.respond_to?(:retired?) && related.retired? }` like we do in the `Notifiable` concern.

Reply:
Comments edited after remembering you had added a comment to the issue mentioning the same things I was mentioning.

Reply:
all fixed :)
(hopefully with Hound too :D)